app/src/org/commcare/android/view/SquareButton.java

```diff
@@ -27,7 +27,12 @@ protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
         int w = MeasureSpec.getSize(widthMeasureSpec);
         int h = MeasureSpec.getSize(heightMeasureSpec);
 
-        w = Math.max(w, h);
+        if(w == 0 || h == 0) { // this can happen in the home screen gridView, and then we'll have an invisible view
```

Comment:
Please put a space in between 'if' and the paren. 'if' is not a function, but rather a keyword, so we shouldn't be using functional call syntax on it.